### PR TITLE
add me game (fight big parma)

### PR DESCRIPTION
**Pull request by jakempock**
# Your checklist for this pull request

## About your game

What is your game about? 
It is a tower defense game about fighting big wheels of parmesan cheese

How to play your game?
WASD - move focus
I - buy menu (buy towers on the focused tile)
K - kill focused tower or damage parmesan

## Code
Check off the items that are true. 
- [x] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. 

> Thanks for PR!

### CODE REVIEW

File: games/fight_big_parma.js [hunk 1]
```diff
@@ -0,0 +1,637 @@
+/*
+@title: fight_big_parma
+@author: me, myself, and that guy
```
Comment (LucasHT22) [inline games/fight_big_parma.js:3]: `@author: me, myself, and that guy`
haha, are you sure?

**Review by LucasHT22 — CHANGES_REQUESTED** [review]

File: games/fight_big_parma.js (hunk 1 above)
Comment (jakempock) [reply games/fight_big_parma.js:3]: quite

Comment (jakempock) [reply games/fight_big_parma.js:3]: Is that a "are you really sure" or "please change this"

Comment (jakempock) [reply games/fight_big_parma.js:3]: "jakempock" isn't my real name or primary github anyway, so it doesn't really matter to me

**Comment by jakempock** [comment]
holy cow thats a lot of commits

**Comment by LucasHT22** [comment]
Sorry about that! we are solving it

File: games/fight_big_parma.js (hunk 1 above)
Comment (LucasHT22) [reply games/fight_big_parma.js:3]: It's because it's not your nickname or your name

**Comment by LucasHT22** [comment]
what did you do to resolve the commits?

**Comment by LucasHT22** [comment]
about the game itself, great game!

**Comment by jakempock** [comment]
I pulled the commits with pull.rebase=true and cherry picked my game commits, also thanks :D

File: games/fight_big_parma.js (hunk 1 above)
Comment (jakempock) [reply games/fight_big_parma.js:3]: alright, changing now

**Review by LucasHT22 — APPROVED** [review]
LGTM! Congrats:)
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

Hack Clubbers would love to see this! Just post a link and 1-2 sentences talking about your game to our [#ship](https://hackclub.slack.com/archives/C0M8PUPU6) [channel](https://hackclub.com/slack/):)

**Pull request merged**